server/subscribe.go

```diff
@@ -684,7 +684,7 @@ func (p *Plugin) hasPermissionToManageSubscription(instanceID types.ID, userID,
 
 	switch cfg.RolesAllowedToEditJiraSubscriptions {
 	case "team_admin":
-		if !p.client.User.HasPermissionToChannel(userID, channelID, model.PermissionManageTeam) {
+		if !p.client.User.HasPermissionTo(userID, model.PermissionManageTeam) {
```

Comment:
Why this change and we are not sending any team ID here. How are we checking it?

Reply:
reverted